language/de-DE/de-DE.lib_joomla.ini

```
@@ -310,12 +309,16 @@ JLIB_FORM_BUTTON_SELECT="Auswählen"
 JLIB_FORM_CHANGE_IMAGE="Bild bearbeiten"
 JLIB_FORM_CHANGE_IMAGE_BUTTON="Bild-Button bearbeiten"
 JLIB_FORM_CHANGE_USER="Benutzer auswählen"
+JLIB_FORM_CONTAINS_INVALID_FIELDS="Das Formular kann nicht abgeschickt werden, da erforderlichen Daten fehlen.<br>Bitte die markierten Felder korigieren und erneut versuchen."
```

```suggestion
JLIB_FORM_CONTAINS_INVALID_FIELDS="Das Formular kann nicht abgeschickt werden, da ihm die erforderlichen Daten fehlen.<br>Bitte die markierten Felder korrigieren und erneut versuchen."
```

Haben wir im Install so drin.